tests/e2e/tutorials/tutorialBase.js

```diff
@@ -332,6 +332,28 @@ class TutorialBase {
     await this.takeScreenshot("checkNodeResults_after");
   }
 
+  async checkResults2(fileNames) {
+    try {
+      await this.takeScreenshot("checkResults_before");
+      const files = await this.__page.$$eval('[osparc-test-id="FolderViewerItem"]',
+        elements => elements.map(el => el.textContent.trim()));
+      assert(files.length === fileNames.length, 'Number of files is incorrect')
```

Comment:
ok, so this was failing in the kember test because:
the kember is a jupyter notebook from the old times, and it sometimes generate an additional file (there is a hook in the notebook that triggers saving the state from time to time). When that additional file is present it does not mean that it failed, just that the timing was a bit different. seeing this assertion means that it will still fail when it happens. I see the point in checking it, but maybe you could make it optional with default to true?

Reply:
done